llvm/test/CodeGen/RISCV/rvv/addi-scalable-offset.mir

```diff
@@ -48,12 +48,14 @@ body: |
     ; CHECK-NEXT: $x10 = ADDI killed $x10, -2048
     ; CHECK-NEXT: $x10 = ADDI killed $x10, -224
     ; CHECK-NEXT: VS1R_V killed renamable $v8, killed renamable $x10
-    ; CHECK-NEXT: $x2 = frame-destroy ADDI $x8, -2048
-    ; CHECK-NEXT: $x2 = frame-destroy ADDI killed $x2, -224
-    ; CHECK-NEXT: $x2 = frame-destroy ADDI $x2, 240
+    ; CHECK-NEXT: $x2 = frame-destroy ADDI $x8, -2032
```

Comment:
Is it possible to split the patch so that we get the changes to ADDIs here without the new CFI instructions? It's really hard to review all of the tests to see if anything changes other than CFI directives.

Reply:
addressed